### DISCUSSION TITLE
IR Toggle display?

### DISCUSSION THREAD

**Discussion by sgrasley** [nealcarney/ESPHome-MaxxFan-controller#4] [Ideas]
Hello,   was wondering if you ever played around with turning off the Houghton  display... I am looking for the IR codes to toggle the display LED.

**Comment by nealcarney** [comment]
Yes, but it never worked well for me.  My understanding is that when the Houghton remote broadcasts a code, it is broadcasting all the settings at once, including the display on/off status.  I tried memorizing them that way (both with display on and off).  If you look at the file Houghton IR codes.xlsx you'll see I have a column D for the display status.  I tried memorizing them that way (both with display on and off).  But when I played back the codes, it didn't seem to change the display status.  Never got a chance to dig in and figure out what wasn't working.

When I did the original work, I started a deep dive of the IR codes using https://github.com/crankyoldgit/IRremoteESP8266, specifically IRRecvDump.  I found that the Houghton was using a protocol that matched CARRIER_AC128.  I was able to decipher most of it, including the position of the display on/off.  But I never finished figuring out the checksums, etc.  Would have been nice to fully figure it out and provide the info so that the guy that is maintaining that repository could add it to his list of devices.
